lib/lambda/update/submitNOSO.ts

```diff
@@ -26,18 +26,28 @@ interface submitMessageType {
   status: string;
   submitterEmail: string;
   submitterName: string;
+  submissionDate: string;
+  proposedDate: string;
   adminChangeType: string;
   stateStatus: string;
   cmsStatus: string;
 }
 
+const convertStringToTimestamp = (date: string) => {
+  const formatedDate = new Date(date).getTime();
+  if (isNaN(formatedDate.valueOf())) throw new Error("Not a valid time");
```

Comment:
Not a big deal but I don't think calling .valueOf() is needed since .getTime() returns a number.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime